website/docs/reference/feature-toggles.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Feature Toggles
+---
+
+Feature toggles are the central concept that we built Unleash around. In Unleash, feature toggles are organized within [projects](../user_guide/projects.md). Feature toggles can have different [activation strategies](../user_guide/activation-strategies.md) for each of their project's [environments](../user_guide/environments.md), and can also be given [variants](../advanced/feature-toggle-variants.md) to facilitate A/B testing.
+
+## Configuration options
+
+Each feature toggle has the following configuration options
+
+| Option | Required? | Default value | Description |
+| --- | --- | --- | --- |
+| name | Yes | N/A | The feature toggle's name. Must be URL-friendly according to [section 2.3 of RFC 3986](https://www.rfc-editor.org/rfc/rfc3986#section-2.3) and must be unique within your Unleash instance. |
+| feature toggle type | Yes | Release | The [feature toggle's type](../advanced/feature-toggle-types.md). |
+| project | Yes | Your current project | The [project](../user_guide/projects.md) that should contain the feature toggle. |
+| description | No | N/A | A short text to describe the feature toggle's purpose. |
+| enable impression data | Yes | No | Whether to enable [impression data](../advanced/impression-data.md) for this toggle or not. |
+
+## Environments
+
+You will probably not want to use the same configuration to enable a toggle in development as you do in production. That's why feature toggles have different activation strategy configurations for each environment.
+
+You can enable and disable a toggles environments independently. You can only choose environments that are active for the current project. When you disable a toggle's environment, it will always evaluate to false in that environment. When you enable an environment, the toggle will evaluate to true or false depending on its activation strategies.
+
+Refer to the [documentation on environments](../user_guide/environments.md) for more details.
+
+## Activation strategies
+
+To enable feature in an environment, you must assign it at least one activation strategy. A feature toggle's activation strategies determine whether it gets enabled for a particular user or instance of an application. When using multiple strategies in a single environment, it is enough that a single strategy evaluates to true for the toggle to get enabled for a user. Whenever a toggle is checked, Unleash will evaluate all the strategies in the current environment.
```

Comment:
`A feature toggle's activation strategies determine whether it gets enabled for a particular user or instance of an application.`

I don't understand `instance of an application`, maybe there is something else we can use here?

Reply:
Great point. I reckon we can simply remove "instance of an application". I was just trying to cover cases where it's not necessarily based on the end user, but maybe it doesn't matter. 

Optionally, we could use "Unleash context" here. That'll cover everything. Thoughts? @gardleopard @sighphyre